s2stac/src/main/scala/crawler/Crawler.scala

```diff
@@ -0,0 +1,484 @@
+        List(
+          StacLink("../catalog.json", StacLinkType.Parent, Some(`application/json`), None),
+          StacLink("../../../../../collection.json", StacLinkType.StacRoot, Some(`application/json`), None)
+        )
+      )
+      printInfo[F](s"month ${monthHistory.toID} did not exist!") map { _ =>
+        CrawlerState.monthHistory(monthHistory, monthCatalog) `combine` CrawlerState.yearCatalogChild(
+          YearCatalogId(monthHistory.year.toID),
+          monthHistory
+        )
+      }
+    }
+
+  private def makeDayParent(history: CrawlerState, dayHistory: History.DayHistory) =
+    if (history.days.contains(dayHistory)) {
+      printInfo[F](s"day ${dayHistory.toID} already exists") *>
+        Sync[F].pure(CrawlerState.initial)
+    } else {
+      val dayCatalog = StacCatalog(
+        "1.0.0-beta1",
+        Nil,
+        s"catalog-day-${dayHistory.toID}",
+        None,
+        s"Catalog holding descendants of day ${dayHistory.day}",
+        List(
+          StacLink("../catalog.json", StacLinkType.Parent, Some(`application/json`), None),
+          StacLink("../../../../../../collection.json", StacLinkType.StacRoot, Some(`application/json`), None)
+        )
+      )
+      printInfo[F](s"day ${dayHistory.toID} did not exist!") map { _ =>
+        CrawlerState.dayHistory(dayHistory, dayCatalog) `combine` CrawlerState.monthCatalogChild(
+          MonthCatalogId(dayHistory.month.toID),
+          dayHistory
+        )
+      }
+    }
+
+  private def getItemAssets(tileInfo: L1CTileInfo): Map[String, StacItemAsset] = {
+    Map(
+      s2Bands
+        .map({ (band: Band) =>
+          band.name.value -> StacItemAsset(
+            s"s3://${l1cBucket}/${tileInfo.path}/${band.name}.jp2",
+            band.commonName map { _.value },
+            None,
+            Set(StacAssetRole.Data),
+            Some(`image/jp2`)
+          ).addExtensionFields(
+            EOAssetExtension(NonEmptyList.one(band))
+          )
+        })
+        .toList: _*
+    )
+  }
+
+  private def makeItem(productInfo: L1CProductInfo, tileInfo: L1CTileInfo): StacItem = {
+    val reprojectedDataGeom = tileInfo.tileDataGeometry.geom.reproject(tileInfo.tileDataGeometry.crs, LatLng)
+    val dataExtent          = reprojectedDataGeom.extent
+    val eoExt               = EOItemExtension(s2Bands, Some(tileInfo.cloudyPixelPercentage.value))
+    StacItem(
+      s"${productInfo.id.value}",
+      "1.0.0-beta1",
+      Nil,
+      geometry = reprojectedDataGeom,
+      bbox = TwoDimBbox(dataExtent.xmin, dataExtent.ymin, dataExtent.xmax, dataExtent.ymax),
+      links = List(
+        // the parent will always be in ../catalog.json, regardless of how we got here
+        StacLink(
+          "../catalog.json",
+          StacLinkType.Parent,
+          Some(`application/json`),
+          None
+        )
+      ),
+      assets = getItemAssets(tileInfo),
+      collection = None,
```

Comment:
I think this should be filled in since it's the name of the collection and also add a link to the collection as well.

Reply:
You're correct!

Reply:
3c9e9eb

Reply:
✨